cmd/collector/app/builder/builder_flags.go

```diff
@@ -27,14 +27,14 @@ import (
 const (
 	collectorQueueSize            = "collector.queue-size"
 	collectorNumWorkers           = "collector.num-workers"
-	collectorPort                 = "collector.port"
-	collectorHTTPPort             = "collector.http-port"
-	collectorGRPCPort             = "collector.grpc-port"
```

Comment:
Usually we don't completely drop support for old flags, we can still accept them (although more code to maintain)